DSE_5.1_Release_Notes.md

```diff
@@ -1,6 +1,25 @@
 # Release notes for DataStax Enterprise 5.1
 DSE 5.1.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any. Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.
 
+# Release notes for 5.1.41
+12 October 2023
```

Comment:
This date is wrong!

Reply:
Thank you for noticing it! As we found out, the dates are taken from jira releases which are generated by the release job, it turns out that logic does not generate dates according to our expectations.